.github/workflows/update-tree-sitter-bash-wasm-file.yml

```diff
@@ -0,0 +1,12 @@
+name: Update Tree Sitter Bash WASM File
```

Comment:
Just to understand, why did you split this into new jobs?
To make the review process simpler? I can understand that if one package creates a regression, it's easier to debug and fix this way.

Reply:
I don't have much experience with github workflows, I just happened to try it like this.

One thought I had was that maybe we'd like to schedule both workflows on different times to avoid breaking things while updating the wasm files in two separate PR, but thing is it is unlikely to happen. In fact, now that I am thinking of it, it would probably make sense to put both of them in the same github workflow.

I could easily combine `update-tree-sitter-bitbake-wasm-file.yml` and `update-tree-sitter-bash-wasm-file.yml` together, but do you think there's a way to combine the three "update-tree-sitter" github workflows into one single workflow (without duplicating code)?

Reply:
I think it's preferable to have multiple workflows. If one update breaks, it's easier to manage. Code duplication is already trimmed down enough in the current state of this PR. I think it can be merged as it is.